vhdbuilder/packer/test/windows-vhd-content-test.ps1

```diff
@@ -179,6 +228,58 @@ function Test-FilesToCacheOnVHD
 
 }
 
+function Test-ValidateAllSignature {
+    foreach ($dir in $map.Keys) {
+        Test-ValidateSinglePackageSignature $dir
+    }
+}
+
+function Test-ValidateSinglePackageSignature {
+    param (
+        $dir
+    )
+
+    foreach ($URL in $map[$dir]) {
+        $fileName = [IO.Path]::GetFileName($URL)
+        $dest = [IO.Path]::Combine($dir, $fileName)
+
+        if(![System.IO.File]::Exists($dest)) {
+            Write-ErrorWithTimestamp "File $dest does not exist"
+            $invalidFiles = $invalidFiles + $dest
+            continue
+        }
+
+        $installDir="c:\SignatureCheck"
+        New-Item -ItemType Directory $installDir -Force | Out-Null
+        if ($fileName.endswith(".zip")) {
+            Expand-Archive -path $dest -DestinationPath $installDir -Force
+        } elseif ($fileName.endswith(".tar.gz")) {
+            tar -xzf $dest -C $installDir
+        }
+
+        $NotSignedList = (Get-ChildItem -Path $installDir -Recurse -File -Include "*.exe", "*.ps1", "*.psm1" | ForEach-object {Get-AuthenticodeSignature $_.FullName} | Where-Object {$_.status -ne "Valid"})
```

Comment:
dll?

Reply:
Print all files in the package in the logs so we can check whether some new types are needed to be signed?

Reply:
Add the log output.